### ISSUE TITLE
Cannot change password after enabling integration

### ISSUE THREAD

**Issue by marcominiluca** [oyvindwe/connectlife-ha#159]
Hi, i need to modify the password. Could you please help me to understand where the password is stored?

**Comment by oyvindwe** [comment]
Oh, I should add a config flow for that!

Currently you need to remove and add the integration to login with a new password. All entities should be preserved though.

**Comment by marcominiluca** [comment]
I tried but i get this error when i put my username and password
![image](https://github.com/user-attachments/assets/97efdee1-98f3-4047-b66d-90f179b4308a)

**Comment by oyvindwe** [comment]
Have you updated to 0.20.2?

**Comment by marcominiluca** [comment]
Yes

**Comment by oyvindwe** [comment]
You can enable debug logging and check the login response for an error message

**Comment by marcominiluca** [comment]
How can I enable it if i removed the integration? :)
![image](https://github.com/user-attachments/assets/2533cab3-19cd-450a-b2c6-32ff99c2c9d2)

**Comment by marcominiluca** [comment]
Yes, I does open the credential pane. I can't enable the debugging log from there

**Comment by oyvindwe** [comment]
Ah, then you need to edit `configuration.yaml` and restart HA:
```
logger:
  default: info
  logs:
    custom_components.connectlife: debug
    connectlife: debug
```

**Comment by marcominiluca** [comment]
Thank you. Here is the status.. seems related to [Issue #157](https://github.com/oyvindwe/connectlife-ha/issues/157)

What do you think?

`2024-11-13 14:40:02.596 DEBUG (MainThread) [connectlife.api] response: {
  "callId": "818cb1d26b49431ab68ca91f3cc7b591",
  "errorCode": 206001,
  "errorDetails": "Missing required fields for registration: preferences.terms.connectlife_terms_conditions.isConsentGranted, preferences.privacy.connectlife_privacy_policy.isConsentGranted",
  "errorMessage": "Account Pending Registration",
  "apiVersion": 2,
  "statusCode": 206,
  "statusReason": "Partial Content",
  "time": "2024-11-13T13:40:02.568Z",
  "registeredTimestamp": 1708423801,
  "UID": "76206a593a1b4587ac822f7696402e63",
  "UIDSignature": "/pTcnT5mnoILw38Et/GHNgiW4po=",
  "signatureTimestamp": "1731505202",
  "created": "2024-02-20T10:10:01.361Z",
  "createdTimestamp": 1708423801,
  "isActive": true,
  "isRegistered": true,
  "isVerified": true,
  "lastLogin": "2024-11-13T13:40:02.512Z",
  "lastLoginTimestamp": 1731505202,
  "lastUpdated": "2024-11-13T11:18:29.925Z",
  "lastUpdatedTimestamp": 1731496709925,
  "loginProvider": "site",
  "oldestDataUpdated": "2024-02-20T10:10:01.361Z",
  "oldestDataUpdatedTimestamp": 1708423801361,
  "profile": {
    "firstName": "NAME",
    "lastName": "SURNAME",
    "city": "N/A",
    "country": "IT",
    "email": "MAIL",
    "zip": "N/A"
  },
  "registered": "2024-02-20T10:10:01.879Z",
  "socialProviders": "site",
  "verified": "2024-02-20T10:10:18.764Z",
  "verifiedTimestamp": 1708423818764,
  "newUser": false,
  "regToken": "st2.s.AtLt........",
  "sessionInfo": {
    "expires_in": "3600"
  }
}`

**Comment by oyvindwe** [comment]
You need to accept the new terms in the app

**Comment by marcominiluca** [comment]
I'd do that but the app enters in a loop when i accept the terms. Also tried with another phone. I hope they release a new version asap

**Comment by CMNTDP** [comment]
> I'd do that but the app enters in a loop when i accept the terms. Also tried with another phone. I hope they release a new version asap
andrdoid 
same problem here... android app enter in loop accepting GDPR terms

**Comment by oyvindwe** [comment]
Fixed in PR #437 - when authentication fails due to password change, you will be asked to log in again. To trigger immediately (and not when refresh token expires), reload the integration.
